bounties/level-2/federation-server.md

```diff
@@ -0,0 +1,109 @@
+    - allow a logged-in user to add, change or remove their stellar account-id
+    - presents the user with their full federation-address consisting of their username and the domain (e.g. `@tyvdh*quest.stellar.org`)
+    - allow logged-id user to delete their account
+  - serves the `/federation` endpoint that 
+    - supports at least the `name` query-type (see [federation protocol])
+- Documentation that 
+  - lays out the installation procedure
+  - lays out the deployment process
+  - details what query-types are supported by the `/federation` endpoint 
+
+
+#### Bounty Description
+
+To successfully hunt this bounty implement and publish a federation-server that allows your friends so sign-up with
+their social-media account so you can start sending XML (or else) to your easy-to remember federation-addresses.
+
+In order for your federation-server to be testable you need to also host a `/.well-known/stellar.toml` file
+on the domain you want to federate. You may use github pages to serve to `toml`-file. Just make sure to configure
+the federation server to match the github-pages domain.
+
+
+#### Review Criteria
+
+The reviewer needs to sign-up with the federation server and configure/add their account-id.  
+The reviewer needs to query the server's `/federation` endpoint with all the query-types described in the documentation.  
+The reviewer may chose the tooling of their choice (e.g. `curl` or the [js SDK](https://stellar.github.io/js-stellar-sdk/FederationServer.html))
+and verify the correct response according to the [federation] server documentation. The reviewer may ask a second person to sign up with their
+account to cross-verify.
+The reviewer should remove/alter their account-id and verify that the reponse changes.
+The reviewer should delete their account and verify that login does not work anymore.
+Depending on the ID-provider the reviewer may on the ID-provider's side revoke access and check that
+login actually stops working on the federation-server UI.
+
+#### Submission Procedure
+
+Implement the deliverables described above and add to the bounty-issue
+- a link to the repository (preferably a PR from implementation branch into base branch – even in your repo)
+- a link to the running application
+- the name of the domain that the federation-server is configured to (e.g. quest.stellar.org or <your-user-name>.github.io)
+
+Add the URL of the deployed federation-server to the github-repo's 'about' section.
+
+
+## Links
+
+- [federation] server documentation
+- [reference implementation]
+- [stellarID.io](https://stellarid.io/) has a UI to sign-up
+
+
+## Ideas
+
+The actual server endpoint is just a simple lookup in a key-value store. Before bringing up a whole DB like postgres or MySQL
+you may consider using [cloudflare workers](https://workers.cloudflare.com/). They provide a [key-value interface](https://developers.cloudflare.com/workers/learning/how-kv-works)
```

Comment:
Is this different than their "durable objects"? Last I knew, they required a paid plan to use the durable objects. If they're the same, it might be tricky to require people to have a paid plan

Reply:
No. KV is also available in free plan. I don't know about the purge-time. But for a installation-under-review it should be enough.